cmd/podman/manifest/create.go

```diff
@@ -13,7 +13,7 @@ import (
 var (
 	manifestCreateOpts = entities.ManifestCreateOptions{}
 	createCmd          = &cobra.Command{
-		Use:   "create",
+		Use:   "create [flags] LIST [IMAGE]",
```

Comment:
```suggestion
		Use:   "create [FLAGS] LIST [IMAGE]",
```
Some commands are [flag] others [FLAG] we should be consistent.